src/pages/documentation/platform-assurance/tutorials/validation-summary.md

```diff
@@ -46,34 +41,33 @@ After you have tested your function, you can select **Upload** to save your vali
 | Key | Type | Description |
 | :--- | :--- | :--- |
 | `uuid` | String | Universally unique identifier for the annotation. |
-| `type` | String | Used to denote the type of annotation and is usually the name of the plugin (for example, analytics). |
+| `type` | String | Used to denote the type of annotation and is usually the name of the plugin \(for example, analytics\). |
 | `payload` | Object | Defines the data that should supplement the event. For Adobe Analytics, this is where the post-processed hit data is contained. |
 
 ### Returning Validation Results
 
-To execute the function in the code editor, select **Test**. The function is expected to return an object that contains the following:
+The validation function is expected to return an object that contains the following:
 
 | Key | Type | Description |
 | :--- | :--- | :--- | 
 | `message` | String | The validation message to display in the summary results. |
 | `events` | Array | An array of event uuids to be reported as matched or not matched. |
+| `links` | Array | An array of ValidationResultLink objects {( type: 'doc'|'product', url: String )} to reference documentation and other resources |
 | `result` | String | This is the validation result and is expected to be one of the enumerated strings: "matched", "not matched", "unknown" |
 
-![](./assets/validation-summary/invalid.png)
-
 ## Viewing Results
 
-The left panel displays the results of all enabled validations, including the tested function in the right panel.
+The results of the function will be displayed in the results section just below the code editor. When a validation result is `unknown` or `not matched` and the `events` array has one or more `uuids`, then those events will be highlighted in the timeline with the following colors:
```

Comment:
Let's rephrase this slightly: "The results of the function are displayed in the results section below the code editor. If the validation result is `unknown` or `not matched` and the `events` array has one or more `uuids`, the events will be highlighted in the timeline with the following colors:"